system-variables.md

```diff
@@ -520,7 +513,7 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 ### tidb_capture_plan_baselines <span class="version-mark">New in v4.0</span>
 
-- Scope: SESSION | GLOBAL
+- Scope: GLOBAL
```

Comment:
This behavior change is intentional and only affects master. See: https://github.com/pingcap/tidb/blob/master/docs/design/2021-12-08-instance-scope.md